ComputeExamples.md

```diff
@@ -585,115 +594,137 @@ Now, let's check that we successfully published a algorithm (create NFT + Datato
 ```
 <!--
     assert(computeEnvs, 'No Compute environments found')
+  }).timeout(40000)
 -->
-  
 
-## 10. Consumer starts a compute job
+## 10. Consumer starts a free compute job
 
   ### 10.1 Start a compute job using a free C2D environment
+<!--
     datatoken = new Datatoken(
       consumerAccount,
       (await consumerAccount.provider.getNetwork()).chainId
     )
+-->
 
 let's check the free compute environment
 ```Typescript
-    const computeEnv = computeEnvs[resolvedDatasetDdo.chainId].find(
-      (ce) => ce.priceMin === 0 || isDefined(ce.free)
+    const computeEnv = computeEnvs.find(
+      (ce) =>
+        !ce?.fees || ce.fees.find((fee) => fee.symbol === 'OCEAN' && fee.amount === '0')
```

Comment:
Should we still keep the OCEAN symbol for compute env fee? Besides OCEAN token, do we have other compute env with fee amount 0?

Reply:
i would say keep it , just in case someone is using old components to run tests on, and those return fees in ocean token